rentalPrice.js

```diff
@@ -1,40 +1,34 @@
-// age - age of driver
-// licence - number of full years person holds driving licence
-// clazz - class of the car from 1 (smallest) to 5 (largest) that person wishes to rent
-// acc - has s/he caused any accidents within last year
-// acc2 - has s/he participated (but not caused) in any accidents within last year
-// season - if it is high season or not
-function price(age, licence, clazz, acc, acc2, season) {
+function price(age, timeLicenceOwned, carClass, accidents, inSeason) {
```

Comment:
Overall, the function is quite long and hard to follow. Maybe it make sense to separate it out to smaller functions which each do one thing?